### PR TITLE
Fix CI failure of main branch by install tensorflow<=2.5.2

### LINKED ISSUE #948: [BUG] Errors found in GraphScope standalone tryout
This is a summary of error or bug found by  GraphScope standalone tryout

### PR DESCRIPTION
**Pull request by lidongze0629**
## What do these changes do?

Fixes CI failure with `tensorflow>=2.6.0` 

## Related issue number

Fixes #948

### CODE REVIEW

File: .github/workflows/build-graphscope-wheels-linux.yml [hunk 1]
```diff
@@ -104,7 +104,7 @@ jobs:
         popd
 
         # run test
-        pip3 install pytest tensorflow --user;
+        pip3 install pytest tensorflow==2.6.0 --user;
```
Comment (sighingnow) [inline .github/workflows/build-graphscope-wheels-linux.yml:107]: I suggest `<=`, as we support previous tensorflow release as well.

File: Makefile [hunk 2]
```diff
@@ -1,6 +1,5 @@
 MKFILE_PATH 			:= $(abspath $(lastword $(MAKEFILE_LIST)))
 WORKING_DIR 			:= $(dir $(MKFILE_PATH))
-NUM_PROC                := $( $(command -v nproc &> /dev/null) && echo $(nproc) || echo $(sysctl -n hw.physicalcpu) )
```
Comment (siyuan0322) [inline Makefile:3]: This is for compatibility of MacOS. Why remove this?

Comment (sighingnow) [reply Makefile:3]: It doesn't always work, instead, we leverage `nproc` in `coreutils` package on Mac.

Comment (siyuan0322) [reply Makefile:3]: OK~

File: coordinator/requirements.txt [hunk 3]
```diff
@@ -1,7 +1,7 @@
 cmake>=3.21.2
 daemons
 etcd-distro>=3.5.1
-graphscope-client
+graphscope-client>=0.8.0
```
Comment (sighingnow) [inline coordinator/requirements.txt:4]: I suggest `==` here.

**Pull request merged**